asmrun/spacetime.c

```diff
@@ -186,6 +188,10 @@ static void maybe_reopen_snapshot_channel(void)
 
 extern void caml_spacetime_automatic_save(void);
 
+#ifndef SIGINT
+#define SIGINT -1
```

Comment:
I just copied this from [here](https://github.com/ocaml/ocaml/blob/trunk/byterun/signals.c#L227). You're right it would be better to just not set the signal action in that case.